deployment/network-operator/values.yaml

```diff
@@ -309,14 +386,19 @@ sriovDevicePlugin:
   #     limits:
   #       cpu: "150m"
   #       memory: "100Mi"
+  # -- Optional resources configuration for the ``kube-sriovdp`` container.
```

Comment:
Maybe something similar as the RDMA above?

# -- The following defines the RDMA resources in the cluster.
  # It must be provided by the user when deploying the chart.
  # Each entry in the resources element will create a resource with the provided
  # <name> and list of devices.

Reply:
I updated the description. In practices I didn't succeed in using the `sriovDevicePlugin` fields, so we will have to revisit this I think.

Reply:
Thanks, let's get this in and we will compare with published version